src/main/java/com/coveros/selenified/Selenified.java

```diff
@@ -396,7 +396,7 @@ protected void startTest(Object[] dataProvider, Method method, ITestContext test
         // setup the rest of the browser details
         capabilities.setInstance(invocationCount);
         DesiredCapabilities desiredCapabilities = capabilities.getDesiredCapabilities();
-        desiredCapabilities.setCapability("name", testName);
+        desiredCapabilities.setCapability("name", testName);    //TODO, this isn't coming out as unique for the coveros tests...need to fix this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1135)